### ISSUE TITLE
FR - Touch as a switch

### ISSUE THREAD

**Issue by KyleTeal** [Aircoookie/WLED#3537]
**Is your feature request related to a problem? Please describe.**
My goal is to use a touch pin to set 1 preset on touch, and a different preset on release. 

**Describe the solution you'd like**
Allow for the touch pin to be used in the same way that a switch pin is used. 

**Describe alternatives you've considered**
I tried setting the pin to "switch" but then it does not work. 

**Additional context**
The goal is to set the light color when the touch pin is pressed and then turn off the light when the touch pin is released.

**Comment by blazoncek** [comment]
If you can compile yourself it should be as easy as adding:
```c++
#define BTN_TYPE_TOUCH_SWITCH     9
```
in `const.h`, line 288 and
```c++
    case BTN_TYPE_TOUCH_SWITCH:
```
on line 100 and modifying
```c++
    if (buttonType[b] == BTN_TYPE_SWITCH || buttonType[b] == BTN_TYPE_TOUCH_SWITCH || buttonType[b] == BTN_TYPE_PIR_SENSOR) {
```
line 256 in `button.cpp` file.

Then report if it is working.

**Comment by blazoncek** [comment]
Will be available in 0.15

**Comment by blazoncek** [comment]
Available in 0.15.0 beta 1.
